bundles/AdminBundle/Controller/Admin/MiscController.php

```diff
@@ -169,7 +170,8 @@ public function jsonTranslationsSystemAction(Request $request)
             }
         }
 
-        $response = new Response('pimcore.system_i18n = ' . $this->encodeJson($translations) . ';');
+        $caseInsensitive = $translator instanceof Translator && $translator->getCaseInsensitive() ? "true" : "false";
```

Comment:
boolean or string ?

Reply:
as it is used in the response js code it needs to be a string here

Reply:
why not return true or false as json?

Reply:
that would be a boolean then

Reply:
see my comment around an hour ago :-)

Reply:
In the end it's a string but ok, you're right now json_encode is used.

Reply:
It's a generated javascript code response and not a json so therefore it's a string in the PHP world and a boolean in the json world. ;)

Reply:
now its a boolean in both worlds :=